components/brave_ads/core/internal/sanitize/sanitize_util.cc

```diff
@@ -0,0 +1,17 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/brave_ads/core/internal/sanitize/sanitize_util.h"
```

Comment:
It feels odd that the rust code is a top-level component, but the C++ wrapper is down here in the internals. Does it make sense to move these files to `components/text_sanitize`?

Reply:
`sanitize_util.cc` is not a generic wrapper, it is intended to be a utility file for Brave Ads. Outside of Brave Ads rust code is available through:
`text_sanitize::strip_non_alphanumeric_or_ascii_characters()`